boxoffice/mailclient.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from flask import render_template
+from flask.ext.mail import Message
+from html2text import html2text
+from premailer import transform as email_transform
+from boxoffice.models import Order
+from boxoffice import mail, app
+
+
+def send_invoice_email(order_id, subject="Thank you for your order!"):
+    """
+    Sends an invoice with a PDF attached, to the order's buyer
+    """
+    with app.test_request_context():
```

Comment:
The context for using `test_request_context` here is unclear. Is this function called outside the request context? If so, the with statement should wrap at the top level as well.